projects/plugins/jetpack/extensions/shared/components/upgrade-nudge/style.scss

```diff
@@ -1,13 +1,41 @@
 @import '../../styles/gutenberg-base-styles.scss';
 
-.jetpack-upgrade-nudge__icon {
-	align-self: center;
-	background: var( --color-plan-premium );
-	border-radius: 50%;
-	box-sizing: content-box;
-	color: $white;
-	fill: $white;
-	flex-shrink: 0;
-	margin-right: 16px;
-	padding: 6px;
-}
+.jetpack-upgrade-plan-banner .jetpack-upgrade-plan-banner__wrapper {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	font-size: 14px;
+	height: 48px;
+	background: black;
+	padding: 0 20px;
+	border-radius: 2px;
+	box-shadow: 0 0 1px inset #fff; }
+	.jetpack-upgrade-plan-banner .jetpack-upgrade-plan-banner__wrapper .banner-title,
+	.jetpack-upgrade-plan-banner .jetpack-upgrade-plan-banner__wrapper .banner-description {
+		color: #fff; }
+	.jetpack-upgrade-plan-banner .jetpack-upgrade-plan-banner__wrapper .jetpack-upgrade-plan-banner__title,
+	.jetpack-upgrade-plan-banner .jetpack-upgrade-plan-banner__wrapper .jetpack-upgrade-plan-banner__description {
+		margin-right: 10px; }
+	.jetpack-upgrade-plan-banner .jetpack-upgrade-plan-banner__wrapper .components-button {
+		flex-shrink: 0;
+		line-height: 1;
+		margin-left: auto;
+		height: 28px; }
+		.jetpack-upgrade-plan-banner .jetpack-upgrade-plan-banner__wrapper .components-button.is-primary {
+			background: #e34c84;
```

Comment:
I wonder if we could switch to using studio colors for those colors; it may prove easier to maintain. Something like this:

```diff
diff --git a/projects/plugins/jetpack/extensions/shared/components/upgrade-nudge/style.scss b/projects/plugins/jetpack/extensions/shared/components/upgrade-nudge/style.scss
index d95940b84..921bf6872 100644
--- a/projects/plugins/jetpack/extensions/shared/components/upgrade-nudge/style.scss
+++ b/projects/plugins/jetpack/extensions/shared/components/upgrade-nudge/style.scss
@@ -1,4 +1,5 @@
 @import '../../styles/gutenberg-base-styles.scss';
+@import "~@automattic/color-studio/dist/color-variables";
 
 .jetpack-upgrade-plan-banner .jetpack-upgrade-plan-banner__wrapper {
 	display: flex;
@@ -6,13 +7,13 @@
 	align-items: center;
 	font-size: 14px;
 	height: 48px;
-	background: black;
+	background: $studio-black;
 	padding: 0 20px;
 	border-radius: 2px;
-	box-shadow: 0 0 1px inset #fff; }
+	box-shadow: 0 0 1px inset $studio-white; }
 	.jetpack-upgrade-plan-banner .jetpack-upgrade-plan-banner__wrapper .banner-title,
 	.jetpack-upgrade-plan-banner .jetpack-upgrade-plan-banner__wrapper .banner-description {
-		color: #fff; }
+		color: $studio-white; }
 	.jetpack-upgrade-plan-banner .jetpack-upgrade-plan-banner__wrapper .jetpack-upgrade-plan-banner__title,
 	.jetpack-upgrade-plan-banner .jetpack-upgrade-plan-banner__wrapper .jetpack-upgrade-plan-banner__description {
 		margin-right: 10px; }
@@ -22,13 +23,13 @@
 		margin-left: auto;
 		height: 28px; }
 		.jetpack-upgrade-plan-banner .jetpack-upgrade-plan-banner__wrapper .components-button.is-primary {
-			background: #e34c84;
-			color: #fff; }
+			background: $studio-pink-40;
+			color: $studio-white; }
 		.jetpack-upgrade-plan-banner .jetpack-upgrade-plan-banner__wrapper .components-button.is-primary:hover {
-			background: #eb6594; }
+			background: $studio-pink-30; }
 		.jetpack-upgrade-plan-banner .jetpack-upgrade-plan-banner__wrapper .components-button.is-primary.is-busy {
 			background-size: 100px 100%;
-			background-image: linear-gradient(-45deg, #e34c84 28%, #ab235a 28%, #ab235a 72%, #e34c84 72%); }
+			background-image: linear-gradient(-45deg, $studio-pink-40 28%, $studio-pink-60 28%, $studio-pink-60 72%, $studio-pink-40 72%); }
 
 .jetpack-upgrade-plan-banner.block-editor-block-list__block {
 	margin-top: 0;

```

What do you think?

Reply:
Completely agree 👍